cmd/validate_image.go

```diff
@@ -199,7 +202,7 @@ Use an inline EnterpriseContractPolicy spec
 
 			if data.strict && !success {
 				// TODO: replace this with proper message and exit code 1.
-				return errors.New("success criteria not met")
+				return CM024.CausedBy(errors.New("success criteria not met"))
```

Comment:
Maybe as a future feature, we can do something like: `CM024.New("success criteria not met")`

Reply:
Totally thought of that but I did not want to have this bigger than it already is, also for the `fmt.Errorf` variant.